addons/libs/images.lua

```diff
@@ -0,0 +1,344 @@
+--[[
+    A library to facilitate image primitive creation and manipulation.
+]]
+
+local images = {}
+local meta = {}
+
+windower.prim.saved_images = {}
+local dragged
```

Comment:
This variable isn't currently used. The texts library uses it to expose mouse events on the text boxes. I think it's worth having in this library as well.